tendrl/commons/objects/definition/master.yaml

```diff
@@ -88,6 +101,129 @@ namespace.tendrl:
+              - "Node[]"
+              - TendrlContext.cluster_name
+              - TendrlContext.sds_name
+              - TendrlContext.cluster_id
+              - TendrlContext.integration_id
+              - Cluster.node_configuration
+            optional:
+              - Cluster.public_network
+              - Cluster.cluster_network
+              - Cluster.conf_overrides
+          name: import created cluster
+          run: tendrl.objects.Cluster.atoms.ImportCreatedCluster
+          type: Update
+          uuid: 352f6190-9b37-11e6-950d-a24fc0d9649f
+          help: Import the created cluster
+        PersistClusterDetailsAfterCreate:
+          enabled: true
+          inputs:
+            mandatory:
+              - "Node[]"
+              - TendrlContext.cluster_name
+              - TendrlContext.sds_name
+              - TendrlContext.cluster_id
+              - TendrlContext.integration_id
+              - Cluster.node_configuration
+            optional:
+              - Cluster.public_network
+              - Cluster.cluster_network
+              - Cluster.conf_overrides
+          name: persist cluster additional details
+          run: tendrl.objects.Cluster.atoms.PersistClusterDetailsAfterCreate
+          type: Update
+          uuid: 352f6190-9b37-11e6-950d-a24fc0d9649a
+          help: Persist additional cluster details
+        Create:
+          enabled: true
+          inputs:
+            mandatory:
+              - "Node[]"
+              - TendrlContext.cluster_name
+              - TendrlContext.sds_name
+              - TendrlContext.cluster_id
+              - TendrlContext.integration_id
+              - Cluster.node_configuration
+            optional:
+              - Cluster.public_network
+              - Cluster.cluster_network
+              - Cluster.conf_overrides
+          name: create cluster
+          run: tendrl.objects.Cluster.atoms.Create
+          type: Create
+          uuid: 352f6190-9b37-11e6-950d-a24fc0d9649b
+          help: create cluster
+        ImportCluster:
+          enabled: true
+          inputs:
+            mandatory:
+              - "Node[]"
+              - DetectedCluster.sds_pkg_name
+              - TendrlContext.integration_id
+          name: import cluster
+          run: tendrl.objects.Cluster.atoms.Import
+          type: Create
+          uuid: 452f6190-9b37-11e6-950d-a24fc0d9649c
+          help: Import the cluster
+        CheckClusterAvailable:
+          enabled: true
+          inputs:
+            mandatory:
+              - TendrlContext.integration_id
+          name: check cluster available
+          run: tendrl.objects.Cluster.atoms.CheckClusterAvailable
+          type: Get
+          uuid: 552f6190-9b37-11e6-950d-a24fc0d9649e
+          help: check if cluster is available
+        CheckNodesInExistingCluster:
+          enabled: true
+          inputs:
+            mandatory:
+              - "Node[]"
+              - DetectedCluster.sds_pkg_name
```

Comment:
why do you need "DetectedCluster.sds_pkg_name" for atom "CheckNodesInExistingCluster"